app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i.freeze
+
+  validates :first_name, :last_name, :email, presence: true
+  validates :email, format: { with: VALID_EMAIL_REGEX }, uniqueness: true
```

Comment:
use the constant built into URI in the standard ruby library
`URI::MailTo::EMAIL_REGEXP`